src/solver.cpp

```diff
@@ -699,7 +699,8 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              ++id,
              Primitive::MultiMarginLoss,
              multimarginloss::MultiMarginLossForward{}.SolverDbId());
-
+    Register(registry, ++id, Primitive::Reduce, reduce::AnyForward{}.SolverDbId());
+    Register(registry, ++id, Primitive::Reduce, reduce::AllForward{}.SolverDbId());
```

Comment:
should we insert empty line at here? Please cross-check the latest convention

Reply:
I didnt insert any empty line between the newly added op and the previous op, I dont see any convention says we should insert a space there